third_party/libusb/webport/src/chrome_usb/chrome-login-state-hook-unittest.js

```diff
@@ -118,104 +114,94 @@ function makeTest(fakeProfileType, fakeSessionState, testCallback) {
  */
 function changeSessionState(fakeNewSessionState, propertyReplacer) {
   propertyReplacer.replace(
-      chrome.loginState,
-      'getSessionState', 
-      function(callback) { callback(fakeNewSessionState); });
+      chrome.loginState, 'getSessionState', function(callback) {
+        callback(fakeNewSessionState);
+      });
   goog.array.forEach(sessionStateListeners, function(listener) {
     listener(fakeNewSessionState);
   });
 }
 
+goog.exportSymbol('test_ChromeLoginStateHook_NoApi', function() {
```

Comment:
Starting from here, this file looks weird to me. Maybe I'm just not parsing the git diff correctly, but I believe that with this patch there are two test_ChromeLoginStateHook_NoApi, but no test_ChromeLoginStateHook_DoesNotFilterInSession.

Reply:
Great catch! Seems like an artifact of an unlucky interactive git add :(
Now I've checked out the original version of this file and redid the const-related changes without any unrelated reformatting.